pkg/controller/networkaddonsconfig/testdata/data.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
```

Comment:
can we add a few more examples like daemonset, secret? then he test will just render all and check them all